website/addons/wiki/static/pagedown-ace/Markdown.Editor.js

```diff
@@ -1704,9 +1721,13 @@ var Range = ace.require('ace/range').Range;
 
             buttons.redo = makeButton("wmd-redo-button", getStringAndKey("redo"), "-220px", null);
             buttons.redo.execute = function (manager) { inputBox.session.getUndoManager().redo(); };
+
             makeSpacer(4);
             makeCheckBox("wmd-autocom-toggle", "autocom", "-240px", "Autocomplete");
 
+            makeSpacer(4);

```

Comment:
Please provide a unique number to `makeSpacer`, as it's used to create element id, which should be unique.